lib/schema-compilers.js

```diff
@@ -43,9 +43,11 @@ function ValidatorCompiler (externalSchemas, options, cache) {
     }
   }
 
-  Object.values(externalSchemas).forEach(s => ajv.addSchema(s))
+  for (const schemaId in externalSchemas) {
```

Comment:
wasn't the idea to use `for ... of`?

Reply:
Sorry, you are right, I'm looping the prototype with the `in`

Reply:
This is actually better